Sources/Logging/Strings/IdentityStrings.swift

```diff
@@ -80,7 +84,25 @@ extension IdentityStrings: LogMessage {
         case let .sync_attributes_and_offerings_rate_limit_reached(maxCalls, period):
             return "Sync attributes and offerings rate limit reached:\(maxCalls) per \(period) seconds. " +
             "Returning offerings from cache."
+        case let .login_called_in_cloud_synced_appuserid_mode(oldUserID, newUserID):
+            return """
+            logIn() called with cloud synced appUserIDs enabled. This shouldn't be an issue, but the purchases
+            for the user will be transferred or blocked depending on the configuration in RevenueCat Dashboard.
+            Old user ID: \(oldUserID)
+            New user ID: \(newUserID)
+            More info: https://rev.cat/auth
+            """
+        case let .logout_called_in_cloud_synced_appuserid_mode(oldUserID, newUserID):
+            return """
+            logOut() called with cloud synced appUserIDs enabled. The current user will be logged out, and
+            a new appUserID will be generated.
```

Comment:
This is actually the same that happens in a normal logout right? Maybe we should mention that the iCloud user id has been reset?

Reply:
oops, probably a copy/pasted string that I meant to updaet